.pylintrc

```diff
@@ -150,10 +157,10 @@ function-rgx=[a-z_][a-z0-9_]{2,30}$
 function-name-hint=[a-z_][a-z0-9_]{2,30}$
 
 # Regular expression matching correct method names
-method-rgx=[a-z_][a-z0-9_]{2,30}$
+method-rgx=(([a-z_][a-z0-9_]{2,})|(assert[A-Z][a-zA-Z0-9_]*))$
```

Comment:
I've removed the max characters restriction for methods because we already have a max line length so that enforces some equilibrium between method and parameter names in the signature. But I can restore it if you think it is not OK.